### PR TITLE
fix css stitching for new chrome 78 (bug in chrome)

### PR DESCRIPTION
**Pull request by yuriieasternpeak**

### CODE REVIEW

File: src/positioning/CssTranslatePositionProvider.js [hunk 1]
```diff
@@ -41,19 +41,17 @@ class CssTranslatePositionProvider extends PositionProvider {
...
    * @inheritDoc
    */
-  setPosition(location) {
+  async setPosition(location) {
     ArgumentGuard.notNull(location, "location");
 
-    const that = this;
     this._logger.verbose(`CssTranslatePositionProvider - Setting position to: ${location}`);
 
     const spp = new ScrollPositionProvider(this._logger, this._executor);
```
Comment (astappiev) [inline src/positioning/CssTranslatePositionProvider.js:49]: Why do you need this? This method looks much more complicated than needed.
Check my JS version or [Java one](https://github.com/applitools/eyes.sdk.java3/blob/846213f1b1a54bb096acf0e6f936a1e25a9ceea7/eyes.selenium.java/src/main/java/com/applitools/eyes/selenium/positioning/CssTranslatePositionProvider.java#L44).

**Review by astappiev — CHANGES_REQUESTED** [review]

File: src/positioning/CssTranslatePositionProvider.js (hunk 1 above)
Comment (yuriieasternpeak) [reply src/positioning/CssTranslatePositionProvider.js:49]: A complete change to this method according to java is not related to the task. We need just make a workaround for chrome issue.

Comment (astappiev) [reply src/positioning/CssTranslatePositionProvider.js:49]: Then your changes should go to `EyesWDIOUtils.translateTo` according to your implementation this method moves the page to a position.
So, the fix should be inside the method. Maybe you use it from other places as well and they will require the fix too.

Comment (yuriieasternpeak) [reply src/positioning/CssTranslatePositionProvider.js:49]: moved changes into EyesWDIOUtils.translateTo

**Pull request merged**